metro-bundler-config-yarn-workspaces/index.js

```diff
@@ -29,6 +29,12 @@ module.exports = function getConfig(from, options = {}) {
         options.nodeModules || path.resolve(from, '..', 'node_modules'),
       ].concat(workspaces)
     },
+    getEnableBabelRCLookup() {
+      // Whether Metro should pick up .babelrc configs from host project
+      return typeof options.enableBabelRCLookup === 'boolean'
+        ? options.enableBabelRCLookup
+        : true
```

Comment:
if the value isn't set in options then it's `undefined`, so perhaps `false` is a better default to keep it backwards compatible? would something like `return options.enableBabelRCLookup || false` work for you?

Reply:
> if the value isn't set in options then it's undefined, so perhaps false is a better default to keep it backwards compatible? 

True. But I have a feeling if the `getEnableBabelRCLookup` method isn't defined here (as it was before) the defaults ends up `true`.

> would something like return options.enableBabelRCLookup || false work for you?

If you're OK with the comment above, than yes, I'll do the change asap.